core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -395,8 +395,8 @@ class StandaloneDynamicAllocationSuite
     }
 
     var apps = getApplications()
-    // kill executor 1
-    assert(sc.killExecutor(executors.head))
+    // kill executor 1, and actually nothing to kill
+    assert(!sc.killExecutor(executors.head))
```

Comment:
@andrewor14  The executors.head is assigned beforehand. for example, you have two executor ID {27,28}. Then, the first one(id 27) is killed with replacement. But I guess the newly created executor cannot be with the same ID. After that, you try to kill the header executor (id 27), it should return empty list (since 27 has been in the pendingToRemove list). Am I right?